sfs/array.py

```diff
@@ -423,16 +460,38 @@ def cube(N, spacing, center=[0, 0, 0], orientation=[1, 0, 0]):
 
 
 def sphere_load(fname, radius, center=[0, 0, 0]):
-    """Spherical secondary source distribution loaded from datafile.
+    """Spherical secondary source distribution loaded from the datafile fname.
```

Comment:
I don't really like this.

If you refer to a parameter, you should use some markup, I would suggest `*fname*`.

But actually I don't really like the name `fname`. It's hideous.

I know that's taken from NumPy (https://docs.scipy.org/doc/numpy/reference/generated/numpy.loadtxt.html), but it's bad nonetheless.

I would prefer changing the parameter name to `file`.

Then the tagline could be:

    Spherical secondary source distribution loaded from *file*.

But that's still bad.
This would be better:

    Load spherical secondary source distribution from *file*.

Or drop the markup in this case. Both is possible.

Reply:
changed handling to
Load spherical secondary source distribution from file.